### PR TITLE
Add 404

### LINKED ISSUE #156: 404 Error page

### PR DESCRIPTION
**Pull request by poofichu**
Resolves #156 

CHANGELOG
:memo: Add markdown file for 404

### CODE REVIEW

**Comment by bobbinrobyn** [comment]
:+1:

**Comment by ryanbrownhill** [comment]
:+1: looks good. the content type is empty but I assume this is because 404 is a one off.

**Comment by Snugug** [comment]
@poofichu Did you try and see how this Markdown will render? Generally you can't mix Markdown and HTML together in the same item.

**Comment by poofichu** [comment]
No sir.  I don't know how to do that.

**Comment by poofichu** [comment]
Ryan says he can show me how to do that.

**Comment by Snugug** [comment]
Partner with @britanyponvelle or @ryanbrownhill, they can show you how

**Comment by Snugug** [comment]
There's also https://github.com/IBM-Watson/design-library/wiki/Developing-for-the-Design-Library as a refernce

**Comment by poofichu** [comment]
I should test this by running the virtual box?

**Comment by Snugug** [comment]
@poofichu Yup, but you can always check out http://dillinger.io/ too to see what rendered Markdown will look like.

Basically, I'm recommending you not use Markdown links, bur rather a full `a` tag

**Comment by poofichu** [comment]
Go it.  When it comes to the classes for the span tags for icons, however, I'm just leaving them like they are for now, unless you have another suggestion.  I don't know how we're calling them yet. :cactus:

**Comment by Snugug** [comment]
@poofichu So, like all of our other classes, they can't be real classes, they need to be `library-class`.

We're following the [North styling conventions](http://pointnorth.io/#styling), so it'll be `icon` as our "component" and `next` as our "aspect", so `icon--NEXT`

**Comment by poofichu** [comment]
Yes!!  Perfect.  Working on getting my virtual box up and running now so I can test and fix.

**Comment by poofichu** [comment]
@Snugug would that be weird though to have it be read "Principles. Forward."  @bobbinrobyn just said them all outloud and it sounded silly.

**Comment by Snugug** [comment]
@poofichu Would there be a period? I think having it read `Principles (next)` wouldn't be too weird, and you can do ligatures with parenthesis

**Comment by Snugug** [comment]
@poofichu It can be HTML. Whichever you think would be easier to deal with

**Comment by poofichu** [comment]
@Snugug Okay, now that we have the sanitize option taken out let me see what I can do.

I'll also Google to see what best practice is out there for icon font/accessibility/weird things to avoid

File: language/404.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Principles
+
+Watson design principles provide the foundation for designing excellent cognitive experiences for users.
+
+[Principles <span class="icon-next"></span>](/principles)
+
+### Guidelines
+
+These style, interaction, and branding guides offer detailed recommendations for elements like color, typography, and motion.
```
Comment (kathrynmcelroy) [inline language/404.md]: Can we change the examples to line up with the guides above? Recommending changing this to: "elements like color, motion, and wordmark."

**Comment by poofichu** [comment]
I'm going to change the content to @kathrynmcelroy 's request, the same for the homepage too.

Looks like the removal of the sanitize option worked beautifully:
image

**Comment by Snugug** [comment]
Closed in favor of #227

**Pull request closed without merging**